src/apps/vpn/serveri18n.cpp

```diff
@@ -154,6 +154,13 @@ QString translateItemWithLanguage(const QString& languageCode,
     }
   }
 
+  // Let's use 'en' translation, which is still better than what we obtain from
+  // Guardian.
+  result = s_items.value(itemKey("en", countryCode, cityName));
+  if (!result.isEmpty()) {
```

Comment:
If this is the final fallback how come it can still be empty?

Reply:
because we don't have the 'en' translation for all the city/countries.

Reply:
What is wrong with what we obtain from Guardian?

Reply:
A lot of things are wrong. For instance they say: Osaka instead of Ōsaka.

Reply:
Should we file a follow up to fix that then?

Reply:
that is from Mullvad. There is nothing we can change...